app/src/main/java/org/schabi/newpipe/util/URLHandler.java

```diff
@@ -0,0 +1,124 @@
+package org.schabi.newpipe.util;
```

Comment:
Maybe you could create a subpackage of `util` containing the classes related to sharing, open intents, description url handling, etc... The `util` package has grown too big.

Reply:
Do you have a name for this subpackage? I can't find an appropriate/a coherent one.

Reply:
`open`? Every action contained is related to opening some piece of information with another app (or NewPipe itself). What do you think?

Reply:
external_communication?

Reply:
That's better, probably too long but who cares ;-)
I'd go with this